### DISCUSSION TITLE
4.0.0 alpha 1

### DISCUSSION THREAD

**Discussion by ryanhowdy** [ryanhowdy/fcms#639] [News]
This is the first alpha release of Family Connection 4.0.0.

Please note that while most sections of the site exist, they are far from feature complete.

However you should be able to setup the site, run through the installation process and even register new members.  However, only some very basic functionality currently exists for each section.  Some are more complete than others.

<hr /><em>This discussion was created from the release <a href='https://github.com/ryanhowdy/fcms/releases/tag/4.0.0-alpha.1'>4.0.0 alpha 1</a>.</em>

**Comment by bishu** [comment]
@ryanhowdy unable to find the option to install in the v4.0.0 alpha 1
